_data/releases.tsv

```diff
@@ -46,4 +46,4 @@ Date	Data release	Code version	Sets	Wells	Experiments	Images	Planes	Size (TB)	Fi
 2020-11-19	prod90	0.9.0	15677	1746563		10646676	72846145	192.92555930017602	26.418725	472.513680031
 2020-12-10	prod91	0.9.1	15696	1753475		10708889	73157773	196.00307508509002	26.792054	476.26
 2021-01-27	prod92	0.9.2	17211	1753475		11423912	76708906	215.07339019756503	28.162387	520.03
-2021-02-11	prod93	0.9.3	17318	1786067		11667886	80820758	228.30808922168106	29.373014	534.22
+2021-02-11	prod93	0.9.3	17235	1754195		11434381	79886738	217.9785752377	28.190256	537.61
```

Comment:
Currently running `psql -h $PGHOST -U omero idr -c 'VACUUM VERBOSE ANALYZE;'` in a screen on `prod93b-omeroreadwrite` (after having dumped the DB). Let's see if that helps reclaiming unused DB space in a couple of hours.